service.watchedlist/addon.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.watchedlist"
+	name="WatchedList"
+	version="1.2.1"
+	provider-name="schapplm">
+	<requires>
+		<import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.simplejson" version="2.0.10" />
+		<import addon="script.module.buggalo" version="1.1.3" /> <!-- min. version 1.1.3 for email report -->
```

Comment:
The buggalo addon does not really work any more. At least for Email notification. But since this did really good service debugging my addon, I would like to keep the dependency. A few try-except-clauses are using buggalo and I would have to make changes at lots of places.
Problem is that it seems to not be maintened any more. My [PR](https://github.com/twinther/script.module.buggalo/pull/6) is unresponded for half a year now.
What I could do is merge [my fork of the buggalo-addon](https://github.com/SchapplM/script.module.buggalo) with an update into repo-scripts and take over the maintenance. What do you think?

Reply:
I think that would be great, if the original author is away for too long there are no problems to take over (if original credits are kept). In fact, there are some long going discussions internally to move some of the addons under "Team-Kodi" so multiple persons can be maintainers of those kind of addons.